content/systems/filecoin_markets/onchain_storage_market/storage_deal_states.md

```diff
@@ -30,10 +30,14 @@ The following describes how a deal transitions between its different states.
   - The sector containing the deal has expired. This is triggered by `StorageMinerActorCode._submitPowerReport` which is called whenver a PoSt is submitted. Power associated with the deals in the sector will be lost, collaterals returned, and all remaining storage fees unlocked.
   - The sector containing the active deal has been terminated. This is triggered by `StorageMinerActor._submitFaultReport` for `TerminatedFaults`. No storage deal collateral will be slashed on fault declaration or detection, only on termination. A terminated fault is triggered when a sector is in the `Failing` state for `MAX_CONSECUTIVE_FAULTS` consecutive proving periods.
 
-Given deal states and their transitions, the following are the relationships between deal states and other economic states and activities in the protocol.
+Given the **onchain deal states and their transitions** discussed above, below is a description of the relationships between **off-chain deal states** and other economic states and activities in the protocol. Note that these states below are **off-chain states tracked internally by the markets module**.
 
 - `Power`: only payload data in an Active storage deal counts towards power.
 - `Deal Payment`: happens on `_onSuccessfulPoSt` and at deal/sector expiration through `_submitPowerReport`, paying out `StoragePricePerEpoch` for each epoch since the last PoSt.
 - `Deal Collateral`: no storage deal collateral will be slashed for `NewDeclaredFaults` and `NewDetectedFaults` but instead some pledge collateral will be slashed given these faults' impact on consensus power. In the event of `NewTerminatedFaults`, all storage deal collateral and some pledge collateral will be slashed. Provider and client storage deal collaterals will be returned when a deal or a sector has expired. If a sector recovers from `Failing` within the `MAX_CONSECUTIVE_FAULTS` threshold, deals in that sector are still considered active. However, miners may need to top up pledge collateral when they try to `RecoverFaults` given the earlier slashing.
 
-![Deal States Sequence Diagram](diagrams/deal-payment.mmd)
\ No newline at end of file
+**NOTE: the states in the following code block are off-chain deal states of the Storage Market module.**
```

Comment:
So, this doesn't explain adequately explain what this means and is confusing. I don't want to hold up this PR any more, but this whole page is quite confusing and needs a lot of work. Until we can do a more indepth pass on this, let's just remove the reference to the off-chain deal states, and the link to the Go file.

Reply:
There's really no need to mention the internal statuses of go-fil-markets in the spec. IF it's neccesary, it's only needed because it shows up in the markets libp2p protocol. but I'd like to defer that for now.